src/Sulu/Bundle/PageBundle/Resources/js/views/PageList/PageList.js

```diff
@@ -27,7 +28,7 @@ type Props = ViewProps & {
 @observer
 class PageList extends React.Component<Props> {
     page: IObservableValue<number> = observable.box();
-    locale: IObservableValue<string> = observable.box();
+    locale: IObservableValue<string> = observable.box(userStore.contentLocale);
```

Comment:
Is that necessary? The way I think about this, this will never apply, because the default value will be used.

Reply:
The webspace default value is only used when the current `this.locale` is not available in the current webspace.

Example I have an additional webspace:

```
example2
  nl (default)
  fr
```

Now I go to example2 switch to fr. When I now relogin (removing hash url) it was again `nl` instead of `fr` but if I use userStore.contentLocale as default value it loads correctly the fr locale. So the defaults set by the developer are now only fallbacks if never a contentLocale was set.

Reply:
Okay, it probably also doesn't hurt, I was just curious.